index.html

```diff
@@ -525,5 +525,52 @@ <h2>Privacy Considerations</h2>
     <section>
       <h2>Security Considerations</h2>
     </section>
+
+    <section class="appendix informative">
+<h2>IANA Considerations</h2>
+<section id="media-type">
+  <h2>The <code>application/credential-1.1+json</code> Media Type</h2>
+  <p>
+    This specification registers the <code>application/credential-1.1+json</code> MIME Media Type specifically for
+    identifying a <a href="https://www.rfc-editor.org/rfc/rfc7519#section-4">JWT Claims Set</a> conforming to the Verifiable Credentials 1.1 format.
```

Comment:
```suggestion
    identifying a <a href="https://www.rfc-editor.org/rfc/rfc7519#section-4">JWT Claims Set</a> conforming to the Verifiable Credentials 1.1 format in the `cty` header.
```

Reply:
What if we wanted a media type that identified an actual credential (as opposed to a JWT Claims Set with a credential in it via the "vc" claim) that conformed to the Verifiable Credentials 1.1 format? Is the plan to only use `application/credential+ld+json` for that? Couldn't developers be confused over the naming here if that's the case? I know I'm confused :).

Reply:
My reading of [Use Explicit Typing in RFC 8725](https://www.rfc-editor.org/rfc/rfc8725#section-3.11) is that we should be talking about using the `typ` field to express that a JWT is specifically a VC-JWT, having the "vc" claim in its JWT Claims Set.

This also matches the pattern used in [RFC 8417](https://datatracker.ietf.org/doc/html/rfc8417#section-2.3) with `application/secevent+jwt` which is used in the `typ` field.

It seems that the explicit typing mentioned on the special topic call is around the `typ` field -- and that this media type should be `application/credential-1.1+jwt`?

Reply:
It seems to me that the way to avoid "redundancy" that the media type is a JWT Claims Set is to use `application/example+jwt`. This also follows the media type `+` pattern that indicates that the rules of the `application/jwt` media type apply and `application/example+jwt` is just a more constrained media type, i.e., there will be some kind of claims associated with `example` that must be present.

This works for `application/secevent+jwt` (`events` claim must be present) and it works for `application/credential-1.1+jwt` (`vc` claim must be present).

Why is this incorrect?

Reply:
Is the problem here related to this text in the [VCDM 1.1 spec](https://www.w3.org/TR/vc-data-model/#jwt-encoding):

> typ, if present, MUST be set to JWT.

If so, I think we should discuss what the options are vs. using `cty` in a way that seems confusing to me.

Reply:
@dlongley the full text behind the RFC you cite is indeed focused on `typ`:

> Per the definition of "typ" in [Section 4.1.9 of [RFC7515]](https://datatracker.ietf.org/doc/html/rfc7515#section-4.1.9), it is
>   RECOMMENDED that the "application/" prefix be omitted.  Therefore,
>  the "typ" value used SHOULD be "secevent+jwt".

Here is similar guidance regarding `typ` for DPOP:

- https://datatracker.ietf.org/doc/html/draft-ietf-oauth-dpop#section-12.5

This pull request is focused on `cty`, here are some links that are specific to this use case, demonstrating similar approaches observed in the wild:

- https://openbankinguk.github.io/read-write-api-site3/v3.1.4/profiles/read-write-data-api-profile.html#step-2-form-the-jose-header
- https://www.rfc-editor.org/rfc/rfc7517#appendix-C.2
- https://ec.europa.eu/digital-building-blocks/DSS/webapp-demo/doc/dss-documentation.html#_table_with_all_attributes_per_format_and_class
- https://www.tec.gov.in/pdf/M2M/TEC%2030003.pdf
- https://1passwordstatic.com/files/security/1password-white-paper.pdf
- https://github.com/decentralized-identity/didcomm-messaging/pull/198

Reply:
The use of `cty` appears to be for non-claimsets (and the existing RFC recommends against using it as mentioned, though we have the author of that text recommending we ignore the RFC). Presumably, `cty` will only be present if a JWT is being used -- which implies to me that `typ` could just be used instead.

But, all that being said -- if there's some good reason to use `cty` here, an appropriate mime type would be one that accurately identifies the payload. The payload is not a credential, so that's the main issue I have with the current name and why I can't support it. The payload is a claimset that includes the `vc` (or `vp`?) claim.

If the name is changed to accurately reflect what the payload is, then I can support the name. I can't support this PR until then.

Regarding the whole `typ` / `cty` approach -- I still question it given my reading of various specs ... and I hope for a clearer technical argument for it, but won't stand in the way of it if there are strong advocates that it's the right way to do things with JWTs these days.